src/test/java/org/betonquest/betonquest/utils/UtilsTest.java

```diff
@@ -1,31 +1,24 @@
 package org.betonquest.betonquest.utils;
 
-import org.betonquest.betonquest.BetonQuest;
+import org.betonquest.betonquest.api.logger.util.BetonQuestLoggerValidationProvider;
 import org.betonquest.betonquest.config.Config;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.MockedStatic;
 import org.mockito.Mockito;
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.logging.Logger;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
I see the point with disallowing star imports, but I believe that `Assertions.*` should be explicitly excluded from that rule.